xbmc/interfaces/swig/AddonModuleXbmc.i

```diff
@@ -21,6 +21,13 @@
 %module(directors="1") xbmc
 
 %{
+#if defined(TARGET_WINDOWS) || defined(TARGET_WIN10)
+#  if !defined(WIN32_LEAN_AND_MEAN)
+#    define WIN32_LEAN_AND_MEAN
+#  endif
+#  include <windows.h>
+#endif
```

Comment:
for filesystem, Sleep, sockets. Without the include it breaks with missing bit or redefinition because a struct gets defined here before the proper one in windows.h